dlc2kinematics/mainfxns.py

```diff
@@ -299,11 +299,9 @@ def compute_umap(
 
     # pandas dataframe managing
     df_clean = df.copy()
-    df_clean.columns = df_clean.columns.droplevel(
-        0
-    )  # Drop scorer row to make it easier to navigate in the df table
+
     key2 = ["x", "y"]
-    df_limbs = df_clean.loc[:, (key, key2)]  # Drop likelihood column
+    df_limbs = df_clean.loc[:, pd.IndexSlice[:, key, key2]] # "Drop" likelihood column
```

Comment:
would we rather allow users to condition on pcutoff; i.e., if 2D data they could pass pcutoff >0.4 and only use these points. 

Also, for maDLC projects, we will need to slice differently cc @jeylau - thoughts?

Reply:
Sounds good! Seb and I will bring changes to it 😊

Reply:
Ok so we checked and actually might be trickier than expected:
As we can't have NaNs in UMAP, we would have to go through the likelihood of each keypoint for each frame and if one of these is < pcutoff, we would discard the entire row/frame.

But if one keypoint has in general a very low pcutoff we will lose a lot of frames (I tried with the example dataset from the DEMO and pcutoff really has to be very low (0.0001) based on those keypoints.

What we could propose is to use the pcutoff option if the user only wants to see the embedding of a single keypoint, what do you think?

Reply:
you could not make them NaNs, but 0,0 then they discarded points would cluster together and that cluster could be smartly cut out :) 

>> But if one keypoint has in general a very low pcutoff we will lose a lot of frames (I tried with the example dataset from the DEMO and pcutoff really has to be very low (0.0001) based on those keypoints.

Actually the demo data should have a `pcutoff` of >0.8 to keep the non-occluded data, so this statement is incorrect.

>> What we could propose is to use the pcutoff option if the user only wants to see the embedding of a single keypoint, what do you think?

See my comment above; I don't think 1 keypoint is useful; that defeats the purpose of deeplabcut; let's take this onto basecamp cc @jeylau